src/sdk/objections/service.ts

```diff
@@ -0,0 +1,82 @@
+import { AxiosError, AxiosRequestConfig, AxiosResponse, Method } from "axios";
```

Comment:
I'm not sure about calling this a service if it is caling the api it seems more like a client. Also not so sure about the folder name.

Reply:
this new structure is now mirroring the module structure used ch-sdk-node 
https://github.com/companieshouse/ch-sdk-node/tree/master/src/services/company-officers
https://github.com/companieshouse/ch-sdk-node/tree/master/src/services/lfp
https://github.com/companieshouse/ch-sdk-node/tree/master/src/services/order/basket

this will help to minimise changes when we come to migrate.